Mathlib/NumberTheory/NumberField/Units.lean

```diff
@@ -74,17 +75,21 @@ namespace NumberField.Units
 
 section coe
 
+-- TODO: figure out precise coercion
+instance : CoeHTC (𝓞 K)ˣ K :=
+  ⟨fun x => algebraMap _ K (Units.val x)⟩
```

Comment:
So you want the units to coerce to K, but not the integers?

Reply:
We have flt-regular as a stress test for this, where we use units *a lot*. If bumping mathlib is a lot of trouble I will be back to this.